proto/event.proto

```diff
@@ -50,17 +63,53 @@ message Histogram {
   map<string, string> tags = 5;
 }
 
+message AggregatedSet {
+  string name = 1;
+  repeated string values = 2;
+  google.protobuf.Timestamp timestamp = 6;
+  map<string, string> tags = 7;
+}
```

Comment:
Same here. We could just make this the one set representation, since the non-aggregated version is perfectly well represented by this one.

Reply:
> They seem equivalent

This is intended. I'm trying to make them as similar as possible to see if we could eventually refactor the metric type into something less repetitive and easier to use.

I had also the idea about using ValueKind instead of f64/String values. This will make Sets compatible with others. What do you think?

Reply:
Using `ValueKind` would be very interesting. Overall I've been trying to think about ways that metrics could be more like normal values in our data representation (/cc @a-rodin). For sets I think it could be relatively straightforward.